content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -123,6 +123,8 @@ If the database provider is MySQL, the values are `1` or `0`. However, if the da
 
 ## <inlinecode>$queryRawUnsafe</inlinecode>
 
+<a id="query-raw-unsafe"></a>
```

Comment:
Huh, what is going on here? The generated one does not work properly?

Reply:
They were showing up as broken links. If i dont do the id thing, they still show as broken links

Reply:
Hmmm - the headline should generate `#queryrawunsafe` I think. 
For example: https://deploy-preview-2140--prisma2-docs.netlify.app/docs/concepts/components/prisma-client/raw-database-access#executerawunsafe 

The diff above now adds `#query-raw-unsafe` which we might not actually need any more?

Reply:
Ok so i  did some digging on markdown links. And the problem was the inlinecode element being used meant that we couldnt link to that header, hence me creating the <a id="query-raw-unsafe></a> element. However, i just learned that you can also link to headers the following way, which i tested and works with the inlinecode component:

## <inlinecode>$executeRawUnsafe</inlinecode> {#execute-raw-unsafe}